src/main/java/strategies/impl/AllStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package strategies.impl;
+
+import strategies.SearchStrategy;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class AllStrategyImpl implements SearchStrategy {
+
+    @Override
+    public List<String> search(List<String> stringsList, Map<String, List<Integer>> searchMap, String[] words) {
```

Comment:
What does `stringsList` mean?

Reply:
You never use `searchMap`

Reply:
stringsList renamed to peopleList
searchMap - removed now